### ISSUE TITLE
CRITICAL - module 'alembic' has no attribute 'migration'

### ISSUE THREAD

**Issue by bhulsken** [marcus67/little_brother#166]
With a newer python (3.10.2 vs 3.9.9 for as far as I've tested), I'm getting this:
2022-01-30 10:59:37,254 - root - CRITICAL - module 'alembic' has no attribute 'migration'
2022-01-30 10:59:37,255 - root - ERROR - Stack trace =   File "/usr/lib/python3.10/site-packages/python_base_app/base_app.py", line 594, in main
    app.start()
  File "/usr/lib/python3.10/site-packages/python_base_app/daemon.py", line 178, in start
    self.run(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/python_base_app/base_app.py", line 486, in run
    raise previous_exception
  File "/usr/lib/python3.10/site-packages/python_base_app/base_app.py", line 466, in run
    self.basic_init()
  File "/usr/lib/python3.10/site-packages/python_base_app/base_app.py", line 449, in basic_init
    raise e
  File "/usr/lib/python3.10/site-packages/python_base_app/base_app.py", line 444, in basic_init
    self.prepare_services(p_full_startup=p_full_startup)
  File "/usr/lib/python3.10/site-packages/little_brother/app.py", line 249, in prepare_services
    self.check_migrations()
  File "/usr/lib/python3.10/site-packages/little_brother/app.py", line 180, in check_migrations
    current_version = db_mig.get_current_version()
  File "/usr/lib/python3.10/site-packages/little_brother/db_migrations.py", line 61, in get_current_version
    context = alembic.migration.MigrationContext.configure(self._persistence.get_connection())

2022-01-30 10:59:37,255 - root - INFO - Terminated with exit code 1

I'm not sure, but I think this is due to the MigrationContext not being available under all circumstances. Importing Alembic (perhaps in other modules) with "import alembic" seems to trigger this, while "from alembic.migration import MigrationContext" seems to work fine. I'll test a bit more, and attach a pull request for a fix.

**Comment by marcus67** [comment]
Hi @bhulsken , thanks for reporting this! Actually, I've already come across this error a while ago when I tried to upgrade the alembic package. I decided against it since I did not have time to look into it. Also, LittleBrother usually runs inside a virtual environment when installed with the Debian package. From your stack trace I would deduce that you install the PyPi package directly, correct? Anyway, any help to tackle this is greatly appreciated. If you find a solution I would upgrade the alembic dependency in the package, too.
